bok_choy/web_app_test.py

```diff
@@ -56,6 +109,11 @@ def setUp(self):
         tags = [self.id()]
         self.browser = browser(tags, self.proxy)
 
+        # Needle uses these attributes for taking the screenshots
+        self.driver = self.get_web_driver()
+        self.driver.set_window_position(0, 0)
```

Comment:
This is copy/pasted out of Needle's classSetUp method. This sets the window position to 0,0 right after starting the browser. I'm not really sure why the original author included it but also I don't think it could hurt.

I'm not sure the answer to your question though, other than "however Needle handles it."

That said, I've thought about this a bit more and as a hardening enhancement I will code up a commit that adds a promise that the element exists and is visible before taking the screenshot.